### PR TITLE
Use Spotbugs instead of Findbugs

### PR DESCRIPTION
**Pull request by olsson**
- Spotbugs is the maintained successor to Findbugs.
- The same plugins and configuration works with Spotbugs.
- Spotbugs can inspect classfiles built with jdk 10 (Findbugs cannot)

### CODE REVIEW

**Comment by xiaodong-izettle** [comment]
👍 Just a question, have you tried with some of our projects, and none of those projects report any issues?

File: basepom/pom.xml [hunk 1]
```diff
@@ -30,14 +30,14 @@
...
-        <asm.version>6.2</asm.version>
+        <asm.version>6.2.1</asm.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <buildNumber>${project.version}</buildNumber>
         <checkstyle.version>8.12</checkstyle.version>
         <dependency-check-maven.version>3.3.2</dependency-check-maven.version>
-        <findbugs-maven-plugin.version>3.0.5</findbugs-maven-plugin.version>
-        <findsecbugs-plugin.version>1.7.1</findsecbugs-plugin.version>
+        <spotbugs-maven-plugin.version>3.1.3</spotbugs-maven-plugin.version>
```
Comment (xiaodong-izettle) [inline basepom/pom.xml]: `3.1.6` is the newest one?

**Comment by xiaodong-izettle** [comment]
Another thing, probably we should also update `osspom/pom.xml` also when we are at it? 😄

**Comment by nzroller** [comment]
Did osspom and pushed that too @olsson @xiaodong-izettle . I ran it with `card-refund` which is java 10 and that worked fine. Good to merge maybe?

**Comment by xiaodong-izettle** [comment]
👍

**Pull request merged**